lib/instance.js

```diff
@@ -545,16 +541,16 @@ module.exports = (function() {
                     if (err) throw err;
                   });
                 }).then(function() {
-                  delete options.skip;
+                  options.skip = undefined;
                 });
               }
             }
           });
         }
       }).then(function() {
-        if (!options.fields.length) return this;
-        if (!this.isNewRecord) return this;
-        if (!this.options.include || !this.options.include.length) return this;
+        if (!options.fields.length || !this.isNewRecord || !this.options.include || !this.options.include.length) {
```

Comment:
Hm, there are quite a few lines that are way longer than this one in the same file? I tried to use 200 chars as a max and even that was work while stricten the jshint settings^^.
Right now 115 chars. Do you still prefere to have it on multiple lines?